CONTRIBUTING.md

```diff
@@ -390,10 +390,7 @@ Once your PR is merged:
 
 - Schedule a [Prod build] from the `main` branch.
 - **!!! IMPORTANT !!!**: If you are updating [packages/azure-openapi-validator/autorest], see [this section](#synchronize-with-changes-to-openapi-alps-repository).
-- Once the build is complete, schedule a [Prod npm release] from that build.
-  You may need to get an approval for the release from the appropriate Azure SDK EngSys team members.
```

Comment:
why are we removing the other lines?
Has the process changed?

Reply:
Yes "Prod npm release" is deprecated